### ISSUE TITLE
Slime Blood Bank

### ISSUE THREAD

**Issue by TripleShades** [citadel-station/citadel-main#14728]
Issue reported from  Round ID: 27168 (Citadel Station)

Reporting client version: 513.1533

Blood from Slime crew is not accepted as blood in the blood bank generator for the purposes of converting to synthetic blood.

**Comment by Arturlang** [comment]
are you certain it's not slime jelly, but actually slimepeople blood?

**Comment by TripleShades** [comment]
Yes, I took my own blood as a slime person crew, and tried to convert it.
I thought maybe it was just blood that wasn't in a bag round-start, so I tested that too, but no, other human blood converted fine.
